scripts/sct_label_vertebrae.py

```diff
@@ -394,11 +432,15 @@ def main(args=None):
     fname_seg_labeled = os.path.join(path_output, file_seg + '_labeled' + ext_seg)
     sct.printv('\nGenerate output files...', verbose)
     sct.generate_output_file(os.path.join(path_tmp, "segmentation_labeled.nii"), fname_seg_labeled)
-    sct.generate_output_file(os.path.join(path_tmp, "segmentation_labeled_disc.nii"), os.path.join(path_output, file_seg + '_labeled_discs' + ext_seg))
+    sct.generate_output_file(os.path.join(path_tmp, "segmentation_labeled_disc.nii"),
+                             os.path.join(path_output, file_seg + '_labeled_discs' + ext_seg))
```

Comment:
For now it is not in because I am not sure wether we should create this file and if it is the best way to create it ( because it will not be there if you use a previosu label file), should it be an option or a default ?